perf_dashboard/perf_data/master.csv

```diff
@@ -0,0 +1,105 @@
+StartTime,ActualDuration,Labels,NumThreads,ActualQPS,p50,p90,p99,cpu_mili_avg_telemetry_mixer,cpu_mili_max_telemetry_mixer,mem_MB_max_telemetry_mixer,cpu_mili_avg_fortioserver_deployment_proxy,cpu_mili_max_fortioserver_deployment_proxy,mem_MB_max_fortioserver_deployment_proxy,cpu_mili_avg_ingressgateway_proxy,cpu_mili_max_ingressgateway_proxy,mem_MB_max_ingressgateway_proxy
+2019-12-04T07:27:03.082419483Z,240,cd4d746c_qps_1000_c_64_1024_v2-nullvm_base,64,1000,970,1366,2287,1,4,47,3,6,37,17,23,81
```

Comment:
is this dynamic?

Reply:
In this PR there is no download functionality supported yet. I will add it in a separate PR.